test/FastTests/Issues/RavenDB_19938.cs

```diff
@@ -0,0 +1,247 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.Documents.Indexes.Analysis;
+using Raven.Client.Documents.Operations.Backups;
+using Raven.Client.Documents.Operations.Configuration;
+using Raven.Client.Documents.Operations.Expiration;
+using Raven.Client.Documents.Operations.Refresh;
+using Raven.Client.Documents.Operations.Revisions;
+using Raven.Client.Documents.Operations.TimeSeries;
+using Raven.Client.Documents.Queries.Sorting;
+using Raven.Client.ServerWide;
+using Raven.Client.ServerWide.Operations;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace FastTests.Issues;
+
+public class RavenDB_19938 : RavenTestBase
+{
+    public RavenDB_19938(ITestOutputHelper output) : base(output)
+    {
+    }
+
+    [Fact]
+    public async Task Can_Create_Database_Via_Builder()
+    {
+        using (var store = GetDocumentStore(new Options
+        {
+            CreateDatabase = false
+        }))
+        {
+            store.Maintenance.Server.Send(new CreateDatabaseOperation(builder => builder.Regular(store.Database)));
```

Comment:
I wonder if `Regular()` is the best name. Maybe `Default()` as indication it returns a db record with default settings or `Init()` to indicate the initialization of the builder